server-kotlin/build.gradle

```diff
@@ -26,28 +26,28 @@ dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
     implementation("org.springframework.boot:spring-boot-starter-web")
     implementation "org.springframework.boot:spring-boot-starter-actuator"
-    implementation("com.fasterxml.jackson.module:jackson-module-kotlin")
+    implementation("com.fasterxml.jackson.module:jackson-module-kotlin:2.11.0")
```

Comment:
It's usually better to not specify versions that are specified by Spring Boot. They update them eventually